app/Http/Controllers/ActivitiesController.php

```diff
@@ -16,4 +16,51 @@ public function index(Request $request)
             ->with(compact('activities'));
     }
 
+    public function create(Request $request)
+    {
+
+        return view('activities.create')
+            ->with(compact('request'));
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+
+        $request->validate([
+            'name' => ['required','string'],
+            'points' => ['required','integer'],
+            'type' => ['required','string'],
```

Comment:
Don't know if there is a way but can you enforce a more specific type such as an enumerated type?